java/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -125,23 +125,34 @@ protected RemoteWebDriver() {
   }
 
   public RemoteWebDriver(Capabilities capabilities) {
-    this(getDefaultServerURL(),
-         Require.nonNull("Capabilities", capabilities), true);
+    this(Require.nonNull("Capabilities", capabilities), true);
   }
 
   public RemoteWebDriver(Capabilities capabilities, boolean enableTracing) {
+    this(Require.nonNull("Capabilities", capabilities),
+      enableTracing,
+      ClientConfig.defaultConfig());
+  }
+
+  public RemoteWebDriver(Capabilities capabilities, boolean enableTracing, ClientConfig clientConfig) {
```

Comment:
I dont think we need to enhance the RemoteWebDriver because :
* it's not representing a local driver 
* you can use the RemoteWebDriverBuilder to build a remotewebdriver object

Reply:
Exactly, this is already solved through the `RemoteWebDriverBuilder`